src/app/meshes/views/MeshTrayView.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <RouteView name="mesh-tray-view">
+    <AppView>
+      <template #title>
+        <div class="tray-title-wrapper">
+          <img
+            aria-hidden="true"
+            src="@/assets/images/icon-circles-ext.svg?url"
+          >
+
+          <h2 class="tray-title">
+            <RouterLink
+              :to="{
+                name: 'mesh-detail-view',
+                params: {
+                  mesh: props.name,
+                },
+              }"
+            >
+              <RouteTitle
+                :title="t('meshes.routes.item.title', { name: props.name })"
+                :render="true"
+              />
+            </RouterLink>
+          </h2>
+        </div>
+      </template>
+
+      <EmptyBlock v-if="props.meshInsight === undefined">
+        {{ t('common.collection.tray.empty_title') }}
+
+        <template #message>
+          <p>{{ t('common.collection.tray.empty_message') }}</p>
+        </template>
+      </EmptyBlock>
+
+      <div
+        v-else
+        class="stack"
+      >
+        <div>
+          <h3>{{ t('meshes.routes.item.overview') }}</h3>
+
+          <div class="mt-4 stack">
+            <ResourceStatus
+              :total="props.meshInsight.services.total ?? 0"
+              data-testid="services-status"
+            >
+              <template #title>
+                {{ t('meshes.detail.services') }}
+              </template>
+            </ResourceStatus>
+
+            <ResourceStatus
+              :online="props.meshInsight.dataplanesByType.standard.online ?? 0"
+              :total="props.meshInsight.dataplanesByType.standard.total ?? 0"
+              data-testid="data-plane-proxies-status"
+            >
+              <template #title>
+                {{ t('meshes.detail.data_plane_proxies') }}
+              </template>
+            </ResourceStatus>
+          </div>
+        </div>
+      </div>
+    </AppView>
+  </RouteView>
+</template>
+
+<script lang="ts" setup>
+import EmptyBlock from '@/app/common/EmptyBlock.vue'
+import ResourceStatus from '@/app/common/ResourceStatus.vue'
+import type { MeshInsight } from '@/types/index.d'
+import { useI18n } from '@/utilities'
+
+const { t } = useI18n()
+
+const props = withDefaults(defineProps<{
+  name: string
+  meshInsight?: MeshInsight
+}>(), {
+  meshInsight: undefined,
+})
+</script>
+
+<style lang="scss" scoped>
+.tray-title-wrapper {
+  display: flex;
+  align-items: baseline;
+  gap: $kui-space-30;
+  // Accounts for the absolutely-positioned close button
+  margin-right: calc($kui-space-30 + 24px);
+}
+
+.tray-title {
+  margin-top: 0;
+}
```

Comment:
We should centralize this sort of code elsewhere (so the CSS and some of the markup used here), preferably before we roll this all out. But again, I'd be fine coming back to sort also if necessary.

Centralizing this will prevent us from having to remember to add these things just in these sorts of routes, and if we need to change them we can do so in one place.

Reply:
I’ll touch up on this later. Having more uses will provide better constraints for where these styles can and cannot be and for now the duplication is fine. Easy to change later.

Reply:
Sounds good to me!